### DISCUSSION TITLE
Age at Vital Status - CDE 12306025 (Updated)

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#45] [API Design Discussion]
caDSR Link: [14480965](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14480965%20and%20ver_nr=1)

Definition: "The age (in days) when the follow-up status of the subject was recorded."

Issues with CDE:
Federation members that collect patient age in years will have to perform a conversion to age in days. This conversion will not be exact as the reported age in years could be an integer that does not capture the additional age in days. Since we cannot provide an exact age in days, our reported data will not match the CDE definition.

Agreed Solution as of December 8, 2023:

Federation does it's own harmonization and does not link to a CDE. Harmonization to convert age in years to age in days round((365.25 * Age_In_Years)).
CDEs are being created with definition "approximate age in days". Federation open API will link to this CDE when it is created.
Other Proposed Solutions:

Use a CDE that uses ISO 8601 definition. This will allow for Federation members to report age data in time formats that suit their data.

**Comment by claymcleod** [comment]
I have included this in #49 as a harmonized data element for subjects where the element is not backed by a CDE. As you state, future plans include creating a new CDE conforming to an ISO 8601 date the describes the approximate age—I recommend we close this issue and open a new one when the time comes.

**Reply by GeoffLyle** [reply]
Agreed, we can deprecate this data element when a new CDE is created.

**Comment by GeoffLyle** [comment]
Note that the ongoing issues with this data element are similar to https://github.com/CBIIT/ccdi-federation-api/discussions/37 and #44 

When updating this data element remember to those other data elements.

**Comment by GeoffLyle** [comment]
Per March 6, 2025 meeting:
CDE [14480965](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14480965%20and%20ver_nr=1) has been retired.

Suggested that we use CDE [12306025](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=12306025%20and%20ver_nr=1).
Definition: The age in days of the subject when the vitals measurement was taken.

While the definition is worded different, I believe the meaning is the same.  Will review this CDE for adoption in the v2 release.

**Reply by mcrusch** [reply]
So, you're thinking we'd update the documentation to point to the new CDE, but the values would remain the same?

**Reply by jsunny23** [reply]
The proposed CDE looks equivalent to me and represents the "age at vital status" concept. The unit of the CDE is also days like our current CDE, so it shouldn't require any changes in value. The only discrepancy, which exists with the current CDE as well, is that nodes can use approximate days instead of exact days.

**Comment by GeoffLyle** [comment]
Per March 20, 2025 meeting:

Federation members agreed the new CDE ([12306025](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=12306025%20and%20ver_nr=1)) is functionally the same as the current CDE.  No changes will be made to the data being shared.  Changes to the updated CDE will be reflected in the implementation code and documentation.  How the changes will be implemented will be discussed in the API Workgroup.
